config/foreman-answers.yaml

```diff
@@ -1,27 +1,20 @@
-# Format:
```

Comment:
Kafo always adds this header so probably good to keep it in and also add it to katello.